api/reports/views.py

```diff
@@ -0,0 +1,69 @@
+# reports/views.py
+from django.contrib.auth.models import User
+from django.contrib.auth import authenticate
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework import generics, permissions
+from rest_framework.authtoken.models import Token
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import AllowAny
+from rest_framework.status import (
+    HTTP_400_BAD_REQUEST,
+    HTTP_404_NOT_FOUND,
+    HTTP_200_OK
+)
+from rest_framework.response import Response
+
+from .models import Report
+from .permissions import IsAdmin
+from .serializers import ReportSerializer, UserSerializer
+
+@csrf_exempt
+@api_view(["POST"])
+def login(request):
+    """
+    Creates and returns token for the user,
+
+    Args:
+        username (str) : username
+        password (str) : password
+
+    Returns:
+        response : token key and status.
+
+    """
+    username = request.data.get("username")
+    password = request.data.get("password")
+
+    if username is None or password is None:
```

Comment:
conditional statements with is None can be avoided and written simply as `not username`